djs_playground/settings.py

```diff
@@ -117,6 +117,8 @@
 # https://docs.djangoproject.com/en/1.11/howto/static-files/
 
 STATIC_URL = '/static/'
+STATIC_ROOT = os.path.join(BASE_DIR, 'djs_playground/static/')
+# STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'
```

Comment:
Could you remove the commented-out line at 121L

Reply:
I intentionally put it there in order to imply our playground supports storage settings.

However, I will send PR after removing out. Thank you.

Reply:
Thanks! And I saw your PR was successfully built.

Reply:
Please let me handle this. Thanks.